src/account/AccountBalance.ts

```diff
@@ -0,0 +1,7 @@
+export interface IAccountBalanceRequest {
```

Comment:
Why do you prefix your interface with I?

See https://stackoverflow.com/questions/31876947/confused-about-the-interface-and-class-coding-guidelines-for-typescript/41967120#41967120 interesting though on that question